src/components/ec-toaster/ec-toaster.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div
+    v-if="messages.length > 0"
+    class="ec-toaster"
+  >
+    <transition-group
+      ref="list"
+      name="list"
+      tag="ul"
+      class="ec-toaster__list"
+      appear
+      @after-enter="rememberTopItemPositions"
+      @before-leave="rememberTopItemPositions"
+    >
+      <li
+        v-for="message of messages"
+        :key="message.id"
+        ref="items"
+        v-toaster-touch="{ minDistance: 50 }"
+        class="ec-toaster__item"
+        @toaster-touch-remove="$emit('remove', message)"
+      >
+        <ec-alert
+          :type="message.type"
+          :title="message.title"
+          :subtitle="message.subtitle"
+          :dismissable="message.dismissable"
```

Comment:
Can the notification be non dismissable? Like staying in the top right corner forever?